src/resources/dicts/pt-br.edn

```diff
@@ -1,655 +1,838 @@
+ :settings-page/theme-dark "Escuro"
+ :settings-page/theme-system "Sistema"
+ :settings-page/custom-theme "Tema personalizado"
+ :settings-page/export-theme "Exportar tema"
+ :settings-page/show-brackets "Mostrar colchetes"
  :settings-page/spell-checker "Verificador ortográfico"
- :settings-page/disable-sentry "Enviar dados de utilização e diagnósticos para Logseq"
- :settings-page/preferred-outdenting "Ativar dedentação lógica"
- :settings-page/auto-expand-block-refs "Expandir as referências de bloco automaticamente ao aumentar o zoom"
+ :settings-page/auto-updater "Atualização automática"
+ :settings-page/disable-sentry "Enviar dados de uso e diagnósticos para o Logseq"
+ :settings-page/disable-sentry-desc "O Logseq nunca coletará o banco de dados do seu gráfico local nem venderá seus dados."
+ :settings-page/preferred-outdenting "Rebaixamento lógico"
+ :settings-page/preferred-outdenting-tip "O lado esquerdo mostra o rebaixamento com a configuração padrão, e o lado direito mostra o rebaixamento com o rebaixamento lógico ativado "
+ :settings-page/preferred-outdenting-tip-more "→ Saiba mais"
+ :settings-page/show-full-blocks "Mostrar todas as linhas de uma referência de bloco"
+ :settings-page/auto-expand-block-refs "Expandir automaticamente as referências de bloco ao dar zoom"
+ :settings-page/auto-expand-block-refs-tip "Esta opção controla se as referências de bloco são expandidas automaticamente ao dar zoom."
  :settings-page/custom-date-format "Formato de data preferido"
- :settings-page/preferred-file-format "Formato de Arquivo preferido"
+ :settings-page/custom-date-format-warning "Reindexação necessária! Referências de diário existentes serão quebradas!"
+ :settings-page/custom-date-format-notification "Você deve reindexar seu gráfico para que essa mudança tenha efeito"
+ :settings-page/preferred-pasting-file-hint "Quando ativado, colar uma imagem da internet fará o download e inserirá a imagem. Quando desativado, ele colará o link da imagem."
+ :settings-page/preferred-file-format "Formato de arquivo preferido"
  :settings-page/preferred-workflow "Fluxo de trabalho preferido"
- :settings-page/enable-timetracking "Ativar controle de tempo"
- :settings-page/enable-tooltip "Ativar dicas de ferramentas"
- :settings-page/enable-journals "Ativar diários"
- :settings-page/enable-all-pages-public "Ativar todas as páginas públicas ao publicar"
+ :settings-page/preferred-pasting-file "Preferir colar arquivo"
+ :settings-page/enable-shortcut-tooltip "Ativar dica de atalho"
+ :settings-page/enable-timetracking "Rastreamento de tempo"
+ :settings-page/enable-tooltip "Dicas de ferramentas"
+  :settings-page/enable-journals "Journals"
+ :settings-page/enable-all-pages-public "Todas as páginas públicas ao publicar"
  :settings-page/home-default-page "Definir a página inicial padrão"
- :settings-page/enable-block-time "Ativar carimbos temporais nos blocos"
+ :settings-page/enable-block-time "Carimbos de hora dos blocos"
```

Comment:
Great change! The understanding was much better 😄

Reply:
Your translation doesn't make sense in our language (Brazilian Portuguese), I suggest leaving it as it is or even another alternative. The translation of "time" is treated differently, as I can refer to time including date, with date, hour and minutes or just hour and minutes, using the term "hora" limits the translation.

Reply:
Thank you.